content/docs/tutorials/cloud-tutorial.md

```diff
@@ -0,0 +1,562 @@
+* `-k`						– displays the key portion of the key and value pairs.
+* `-A`						– reads from all partitions available.
+* `--smart-module <module>` – runs a [SmartModule](#smartmodules) module on the output, then displays the results.
+
+## An Easy Fluvio Script — Bash
+
+Here is a simple script that pushes timestamped comments to a remote database.
+
+This code generates a message string that contains the current time, and the
+contents of the first argument passed to it.
+
+Once done, it checks to see if the hard coded topic exists. If it
+does, good, it moves on. If not, it calls upon `fluvio topic create`
+to generate the new topic. Then, it makes a call to `fluvio produce`
+and sends the timestamped message off to the Cloud.
+
+{{<code file="code/bash/timekeeper.sh" lang="bash" copy=true >}}
+
+##### To Run the Script
+
+Save the above script to a file – here it is named `timekeeper.sh` – and set it
+as executable.
+
+You can then run the script with:
+
+%copy first-line%
+```bash
+$ ./timekeeper.sh 'I love cupcakes'
+topic "timekeeper" created
+```
+
+Again, you can use `consume` to view the results.
+
+%copy first-line%
+```bash
+$ fluvio consume timekeeper -dB
+Consuming records from the beginning of topic 'timekeeper'
+2022-09-01T18:46:12-07:00 : I love cupcakes
+```
+
+## Connectors
+
+If you wish to automatically collect information from one source and send it to
+Fluvio, Connectors are the way to go. When given the information on the
+interface through the Connector configuration file, Fluvio can poll a multitude of
+input types.
+
+In this tutorial, we will be looking at the [HTTP Connector](/connectors/sources/http/) setup, connecting
+to the `catfacts.ninja` JSON database.
+
+### Connector Config Layout
+
+This is the template YAML connector file. To make it useful, it needs to be
+populated – which we will do in the next step. See
+[the documentation](/connectors/) for the parameters available for use.
+
+%copy%
+```yaml
+# connect.yml
+version:
+name:
+# the name that you will see when listing the connectors
+type:
+# connection type:
+# source connections currently support: HTTP, Kafka, MQTT, Postgres -source
+# sink connections currently support: Dynabodb, Kafka, Postgres, Slack -sink
+topic:
+# the topic to produce to or consume from
+direction:
+# specifies as source or sink
+parameters:
+  # type specific parameters
+  # smart module parameters
+  # see the docs for the list of parameters
+
+# producer:
+#   linger:
+#   batch-size:
+#   compression:
+# optional content for producer type connector
+
+# consumer:
+#   partition:
+# optional content for consumer type connector
+```
+
+Thankfully, filling it out is simple. For any connection, you need a name,
+the connection type, the direction in which it is connecting, and what topic
+to connect to.
+
+For the HTTP-specific parameters you will need to specify the link it is
+polling, and the interval at which it polls.
+
+%copy%
+```yaml
+# connect.yml
```

Comment:
I'm thinking we also extract this into its own file, and embed it with `code` shortcode, since it is complete, and potentially testable. (You don't need to add tests for this at the moment)